frontend/ui/widget/sortwidget.lua

```diff
@@ -445,7 +445,7 @@ function SortWidget:_populateItems()
     end)
 end
 
-function SortWidget:swipeItems(pos1, pos2)
+function SortWidget:moveItemTo(pos1, pos2)
```

Comment:
(Stated that in some previous comment) I think this one, you wanted to name it `swapItems(pos1, pos2)` - and `swapItems` is better than my early suggestion of `moveItemTo`.